src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -8,16 +8,16 @@ namespace Microsoft.CodeAnalysis.CSharp.CodeStyle
     internal static class CSharpCodeStyleOptions
     {
         // TODO: get sign off on public api changes.
-        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CSharpCodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
+        public static readonly Option<bool> UseVarWhenDeclaringLocals = new Option<bool>(nameof(CodeStyleOptions), nameof(UseVarWhenDeclaringLocals), defaultValue: true,
```

Comment:
Post the refactoring, feature name isn't used for anything short of people explicitly filtering options by that. I know there's some code in one of the analyzer services I've seen somewhere that uses feature name to filter for "if this option with this feature name changed, recompute something" but that's it.